cpp/src/Slice/Parser.cpp

```diff
@@ -64,10 +64,12 @@ Slice::CompilerException::reason() const
     return _reason;
 }
 
+// Forward declare things from Bison and Flex the parser can use.
+extern int slice_parse();
+extern int slice_lineno;
```

Comment:
This is a macro that's equivalent to `yylineno`, the inbuilt line tracker in Flex.